bin/pygrb/pycbc_grb_inj_finder

```diff
@@ -133,6 +136,32 @@ def read_hdf5_triggers(inputfiles, trig_dict, verbose=False):
                 ))
         datasets_n[name] = (shape, dtype)
 
+    def cp_dataset_cont(dataset, xlen, xidentifier, out_dct, pos_full):
+        """Copy dataset contnets into out dictionary.
+
+        Parameters
+        ----------
+        dataset : iteration of datasets_ifo or datasets_n
+
+        xlen : int. The length of IFO identifier (3) or network (8). 
```

Comment:
Given `xidentifier` let the function determine `xlen`, so there is no need to pass `xlen`

Reply:
Addressed in recent file update.